### ISSUE TITLE
Feature Request: Export multiple graphs

### ISSUE THREAD

**Issue by adamlodge** [archesproject/arches#1361]
Would be nice to be able to export all graphs (branches and resource models) all at one time and as separate files for each respective branch and resource model - perhaps all zipped together.

This will make backup of graph development work less time consuming.

It could be a potential configuration to specify resource models only or branches only, and also could be a config to specify individual graphs.  All of this would require additional UI though, and I would call it secondary - especially given that if you download everything as seperate files, you can just delete the files you dont actually care about.

**Comment by ryan86** [comment]
`python manage.py packages -o export_graphs -d /path/to/destination/directory -g graphid, 'branches', 'resource_models', 'all'`

**Comment by adamlodge** [comment]
Looks very good, but two comments...

1> On an early run, for the -g switch, I provided 'resource models' rather than 'resource_models'.  The error message said something about a malformed UUID.  Would be nice if it just said I gave an invalid option in the -g. 

2> I noticed that it exported the default "node" and "node/nodeType" branches.  Maybe those two should not be exported in order to avoid importing them into a system that already has them.
